src/engine.js

```diff
@@ -107,31 +104,17 @@ Engine.prototype.off = function off (event, callback) {
  * and update the internal models with the server response.
  * 
  * Once the response has arrived trigger a 'reload-succes' or 'reload-error' event.
+ * 
+ * @param {string} sourceId - The sourceId triggering the reload event. This is usefull to prevent uneeded requests and save data.
+ * @param {string} forceFetch - ????
  */
-Engine.prototype.reload = function reload () {
-  var params = this._getParams();
+Engine.prototype.reload = function reload (sourceId, forceFetch) {
+  var params = this._getParams(sourceId, forceFetch);
```

Comment:
Do we need to pass `sourceId` and `forceFetch` to this method? It looks like we're not using them.